spec/features/user_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'User session actions', type: :feature do
+  before :each do
+    User.create(name: 'Oscar')
```

Comment:
Creating objects like that in an RSpec test is not a good practice.
Please, use that gem [FactoryBot](https://github.com/thoughtbot/factory_bot_rails)